src/manifest-creation.ts

```diff
@@ -44,7 +44,7 @@ export class ManifestCreation {
       hook_attributes: {
         url: process.env.WEBHOOK_PROXY_URL || `${baseUrl}/`
       },
-      name: manifest.name || pkg.name,
+      name: manifest.name || pkg.name || process.env.PROJECT_DOMAIN,
```

Comment:
What about putting `PROJECT_DOMAIN` first? If you're running on glitch, I think it's more common that you'd want to use the glitch app name rather than whatever was specified in the manifest.

Reply:
I agree. That's what I was going to do before. I swear back then I had what I _think_ was a valid reason for doing so, but I can't quite remember now.

In any case, as @wilhelmklopp suggested, the glitch randomized names are far more useful, so I'll put `PROJECT_DOMAIN` first.